### ISSUE TITLE
Everything is broken

### ISSUE THREAD

**Issue by namlier** [Waavi/translation#51]
Everything is broken with new version

**Comment by nepster-web** [comment]
I confirm. All broke.

**Comment by sildraug** [comment]
Should be fixed on 2.1.4.8. Could you confirm?

**Comment by cirdog** [comment]
I've encountered the "meltdown" after a blind "composer update" before, but later have fixed it somehow.

Just a blind guess:

1) Also remember to delete "/bootstrap/cache/services.php" BEFORE "composer update".
2) Also check own code for the old service-key "UriLocalizer" and replace it with the new key "translation.uri.localizer" everywhere.
3) When these are done, then run composer update and let it php-artisan-optimize on its own.

Laravel's postUpdate hook will call "php artisan optimize" and this will somehow parse the autoloader & stuff in order to generate the services.php, but strangely (I haven't really investigated) the pre-composer-update stuff will interfere with it even though I suppose that it's an over-writing operation...

All blind guesses but since I've fixed the meltdown by blindly deleting services.php, it's worth a try I guess.
